source/exe/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ target_link_libraries(envoy rt)
 target_link_libraries(envoy dl)
 
 include_directories(SYSTEM ${ENVOY_OPENSSL_INCLUDE_DIR})
+include_directories(SYSTEM ${ENVOY_RAPIDJSON_INCLUDE_DIR})
```

Comment:
Removed. It was due to the initial integration of rapidjson. No longer needed now that rapidjson headers are only in the .cc file.